docs/technical_development_guide/source/changelog/changelog_dev.rst

```diff
@@ -6,6 +6,35 @@ Development Changelog
     :version: v3.5.0
     :released: pending
 
+    .. change::
+        :tags: docs, technical reference guide
+        :pullreq: 1361
+
+        **Date**: 2025-03-12
+
+        Title:
+        Update Technical Reference Guide from 3.3.0 to develop
+
+        Description:
+        Update the Technical Reference Guide based on the PRs from the split-off point of ResStock Release 3.3.0 to current develop.
+
+        Assignees: Anthony Fontanini
+    .. change::
+        :tags: workflow
+        :pullreq: 1353
+
+        **Date**: 2025-03-11
+
+        Title:
+        Latest OS-HPXML
+
+        Description:
+        All OpenStudio-HPXML changes, no required ResStock changes. Updates to NEEP ASHP sample files, fix possible HVAC sizing error, HPXML class update for attic/foundation types, Speed up weather processing, combi boiler error fix, smooth shcedule EV plugload, shift all schedules in sync and fix occupancy aggregation.
+
+        OpenStudio-HPXML: `pull request 1918 <https://github.com/NREL/OpenStudio-HPXML/pull/1918>`_, `pull request 1930 <https://github.com/NREL/OpenStudio-HPXML/pull/1930>`_
```

Comment:
These are related to 1347.

Reply:
Addressed: [15778bb](https://github.com/NREL/resstock/pull/1361/commits/15778bb4e6bd4792404aed92129bfeb28c5e3761)